OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -91,9 +69,33 @@ public class AircraftInfo : ITraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 		[Desc("The condition to grant to self while at cruise altitude.")]
 		public readonly string CruisingCondition = null;
 
+		[Desc("Can the actor hover in place mid-air? If not, then the actor will have to remain in motion (circle around).")]
+		public readonly bool CanHover = false;
```

Comment:
Would `Hovers` and `Slides` be more consistent with the other parameter names?

Reply:
Maybe, but since both will go away for good next dev cycle, I feel like renaming `CanHover` now after all these years would just cause unnecessary work for no real benefit, so IMO let's just leave them be until they get replaced by `IdleSpeed`/`SlideRadius` (or whatever we'll end up calling their replacements).

Reply:
Ok, thats fine by me.